src/test/java/com/salesforce/storm/spout/dynamic/coordinator/SpoutCoordinatorTest.java

```diff
@@ -243,61 +243,6 @@ public void testSubmittingNewSpout() throws InterruptedException, ExecutionExcep
         assertEquals("ExecutorService has no running threads", 0, spoutCoordinator.getExecutor().getActiveCount());
     }
 
-    /**
-     * Test what happens when a running spout instance finishes "normally" by
```

Comment:
Both of these tests seem relevant, just slightly changed.  The first should validate that when the spout isComplete returns true, it stops running (perhaps moved under SpoutRunnerTest).  

The 2nd test is validating that Coordinator properly queues VSpout instances, which seems like a valid thing to continue testing.